tests/dummy/app/pods/link/template.hbs

```diff
@@ -120,6 +120,25 @@
       </div>
     </div>
 
+      <div class="example">
+          <div class="title">in-line</div>
+          <div class="demo">
+            {{frost-link 'In-line' 'link.min'
```

Comment:
link-to natively support an in-line format with its own template. By default, when consumer is using an in-line format, first positional param will be set onto the `linkTitle` prop of the link-to component, and then being used in the template. Since frost-link overwrite the default `link-to` template, this thing no longer works.